cmd/cartesi-rollups-cli/root/machine/snapshot.go

```diff
@@ -0,0 +1,47 @@
+// (c) Cartesi and individual authors (see AUTHORS)
+// SPDX-License-Identifier: Apache-2.0 (see LICENSE)
+
+package machine
+
+import (
+	"github.com/cartesi/rollups-node/internal/logger"
+	"github.com/cartesi/rollups-node/internal/machine"
+	"github.com/spf13/cobra"
+)
+
+var Cmd = &cobra.Command{
+	Use:     "save-snapshot",
+	Short:   "Saves a Cartesi Machine Snapshot to the designated folder",
+	Example: examples,
+	Run:     run,
+}
+
+const examples = `# Save the default Rollups Echo Application snapshot:
+cartesi-rollups-cli save-snapshot`
+
+var (
+	sourceDockerImage string
+	tempContainerName string
+	destDir           string
+)
+
+func init() {
+	Cmd.Flags().StringVar(&sourceDockerImage, "docker-image",
+		"cartesi/rollups-machine-snapshot:devel",
+		"Docker image containing the Cartesi Machine snapshot to be used")
+
+	Cmd.Flags().StringVar(&tempContainerName, "temp-container-name", "temp-machine",
+		"Name of the temporary container needed to extract the machine snapshot files")
+
+	Cmd.Flags().StringVar(&destDir, "dest-dir", "./machine-snapshot",
```

Comment:
Since `./machine-snapshot` is the default path, could you add it to the git ignore?

Reply:
Done!